aten/src/ATen/native/native_functions.yaml

```diff
@@ -6772,7 +6772,7 @@
   dispatch:
     CPU, CUDA: ormqr
 
-- func: _lu_with_info(Tensor self, bool pivot=True, bool check_errors=True) -> (Tensor, Tensor, Tensor)
+- func: _lu_with_info(Tensor self, bool pivot=True, bool check_errors=True) -> (Tensor LU, Tensor pivots, Tensor info)
```

Comment:
I guess naming the outputs is upsetting the backcompat check:

```
Aug 02 13:34:01 The PR is introducing backward incompatible changes to the operator library. Please contact PyTorch team to confirm whether this change is wanted or not. 
Aug 02 13:34:01 
Aug 02 13:34:01 Broken ops: [
Aug 02 13:34:01 	aten::_lu_with_info(Tensor self, bool pivot=True, bool check_errors=True) -> (Tensor, Tensor, Tensor)
Aug 02 13:34:01 ]
```

We can tell that check this is OK by extending the list here:

https://github.com/pytorch/pytorch/blob/b58e04f1560de19faf93289eed1da024b914d892/test/backward_compatibility/check_backward_compatibility.py#L28

I wonder if naming the tensors will cause any problems? 

@nikitaved would you serialize a torchscript program that calls `_lu_with_info` using a previous version of PyTorch and verify you can run it as expected with this modification?

Reply:
Confirm, scripting works for the unnamed variant.